app/controllers/application_controller.rb

```diff
@@ -3,8 +3,14 @@
 class ApplicationController < Sinatra::Base
 
   configure do
+    enable :sessions unless test?
+    set :session_secret, "secret"
     set :public_folder, 'public'
     set :views, 'app/views'
   end
 
-end
\ No newline at end of file
+  get '/' do
+    erb :'application/index'
+  end
+
```

Comment:
so we would want to set up some helper methods here to handle `current_user` and `logged_in?` so that we do not have to keep checking session in the controllers 

```ruby
helpers do 

  def logged_in?
    !!current_user 
  end 

  def current_user 
    @current_user ||= User.find_by_id(session[:id])
  end 
end
```